routing/road_graph.hpp

```diff
@@ -54,25 +54,40 @@ inline bool AlmostEqualAbs(Junction const & lhs, Junction const & rhs)
 /// The Edge class represents an edge description on a road network graph
 class Edge
 {
+  // Three combinations are possible:
+  // Real and PartOfReal are raised -- a real edge.
+  // Real is not rised, PartOfReal is raised -- a fake edge that is a part of a real one.
+  // Both not raised -- a fake edge that is not a part of any real segment.
```

Comment:
Why not to create a flag for a fake edge?

Reply:
Real is that flag. If it is not raised an edge is fake.